harbors/schema.py

```diff
@@ -96,18 +96,18 @@ class Meta:
     municipality = graphene.String()
     image_file = graphene.String()
     servicemap_id = graphene.String()
-    zip_code = graphene.String()
+    zip_code = graphene.String(required=True)
```

Comment:
also `phone` and `email`, they are non-null too

Reply:
and `image_link`

Reply:
How bout `suitable_boat_types`?

Reply:
yeah, `suitable_boat_types` should be also required, as it can be empty list, but not `null`

Reply:
Good point gentlemen! Will fix these.